### PR TITLE
Create-mock-for-entity-user

### PR DESCRIPTION
**Pull request by LinconOssa**
The mock is created but there are typing problems and problems in activating and editing that will clear up future tasks that I have already assigned

### CODE REVIEW

**Review by ANGIECATHERINEP — APPROVED** [review]
The request was made correctly

File: src/pages/privileges/outlets/users/types/forms.types.ts [hunk 1]
```diff
@@ -8,6 +8,12 @@ interface IAssignmentFormEntry {
 
 interface IGeneralInformationEntry {
```
Comment (carin91) [inline src/pages/privileges/outlets/users/types/forms.types.ts:9]: This type should be adjusted, deleting the old definitions as id or userId. If you have any trouble of functionality you must resolve it.

File: src/pages/privileges/outlets/users/types/forms.types.ts [hunk 2]
```diff
@@ -8,6 +8,12 @@ interface IAssignmentFormEntry {
...
   id: string;
+  k_Usuari: string;
```
Comment (carin91) [inline src/pages/privileges/outlets/users/types/forms.types.ts:11]: Create a service folder in src/ and locate the interface for the user.

As you can see below, we do this for crediboard:
image

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/users/types/forms.types.ts (hunk 1 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/users/types/forms.types.ts]: Solved according to what was discussed in other tasks to solve the rest

File: src/pages/privileges/outlets/users/types/forms.types.ts (hunk 2 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/users/types/forms.types.ts]: Solved

**Review by carin91 — CHANGES_REQUESTED** [review]
you must create de "linix-users" mock on the DB

**Review by carin91 — APPROVED** [review]

**Pull request merged**